trieste/utils/misc.py

```diff
@@ -16,7 +16,7 @@
 from abc import ABC, abstractmethod
 from time import perf_counter
 from types import TracebackType
-from typing import Any, Callable, Generic, Mapping, NoReturn, Optional, Type, TypeVar
+from typing import Any, Callable, Generic, Mapping, NoReturn, Optional, Tuple, Type, TypeVar
```

Comment:
you shouldn't need Tuple, with `from __future__ import annotations` above you can use `tuple` instead
```suggestion
from typing import Any, Callable, Generic, Mapping, NoReturn, Optional, Type, TypeVar
```